packages/eos-components/src/styles.scss

```diff
@@ -56,7 +56,7 @@ $carousel-indicator-hit-area-height: 10px !default;
 $carousel-indicator-spacer:          3px !default;
 $carousel-indicator-transition:      all .6s ease !default;
 
-@import 'channel-overrides';
+@import "../../template-ui/src/overrides/styles.scss";
```

Comment:
this will work, but we'll have the styles override from the latest channel override, right now I think it's sikana. Maybe we should override that file after the last custom channel is built.

Reply:
Good catch! I just appended a commit with the fix.